modules/machine-health-checks-creating.adoc

```diff
@@ -6,7 +6,8 @@
 [id="machine-health-checks-creating_{context}"]
 = Creating a `MachineHealthCheck` resource
 
-You can create a `MachineHealthCheck` resource for all machine pools in your cluster except the `master` pool.
+You can create a `MachineHealthCheck` resource for all `MachineSets` in your cluster.
+You should not create a `MachineHealthCheck` resource that targets `master` machines.
```

Comment:
Please change "`master`" to "control plane machines". Sorry I should have noticed this before.

Reply:
Fixed